shared/src/packages/sites/libs/validation-schemas/site-create-step-industry.validation-schema.ts

```diff
@@ -0,0 +1,22 @@
+import joi from 'joi';
+
+import { SiteValidationMessage, SiteValidationRule } from '../enums/enums.js';
+import { type SiteCreateIndustryName } from '../types/types.js';
+
+const siteCreateStepIndustry = joi
+  .object<SiteCreateIndustryName, true>()
+  .keys({
+    selectIndustry: joi.string().allow('').messages({
+      'string.empty': SiteValidationMessage.SITE_INDUSTRY_REQUIRE,
+    }),
+    enterIndustry: joi
+      .string()
+      .pattern(SiteValidationRule.SITE_INDUSTRY_REGEX)
+      .messages({
+        'string.empty': SiteValidationMessage.SITE_INDUSTRY_REQUIRE,
+        'string.pattern.base': SiteValidationMessage.SITE_INDUSTRY_WRONG,
+      }),
+  })
+  .or('selectIndustry', 'enterIndustry');
```

Comment:
use `xor` and add empty value for `selectIndustry` and `enterIndustry`

Reply:
@bohdan-degtiar its working but wothout errors messages (empty messages, pattern working.
![image](https://user-images.githubusercontent.com/106602282/229748360-01ab1df1-3638-4fd9-bbdb-33c1e9042f89.png)